src/server.ts

```diff
@@ -0,0 +1,156 @@
+import {db} from "./db-module";
+import {processDataToFormat} from "./data-processor";
+const request             = require('request-promise');
+const R                   = require('ramda');
+
+const USERS_TABLE_NAME: string = "github_users";
+const LIKED_LANGUAGES_NAME: string = "liked_languages";
+
+//Interface for the user objects
+interface GithubUsers{ 
+  id : number,
+  login: string,
+  name: string,
+  location: string,
+  company: string,
+  followers: number
+};
+
+/**Creates the table with name USERS_TABLE_NAME to store users*/
+const createUsersTable = () => {
+  db.none(`CREATE TABLE ${USERS_TABLE_NAME} (id BIGSERIAL PRIMARY KEY, login TEXT UNIQUE, name TEXT, location TEXT,company TEXT, followers INT)`)
+  .catch((e: Error) => {
+    errorHandler(e);
+  })
+  .then(() => process.exit(0))
+}
+
+/**Creates the table with name LIKED_LANGUAGES_NAME to store
+ * the relationship between users and languages they like
+ */
+const createLikedLanguagesTable = () => {
```

Comment:
There is a simpler syntax for adding the reference on `user_id`...
Also, we really prefer that these long lines are split into several lines, to increase readability

Reply:
The next commit will have a simpler syntax for the foreign key definition.
The long lines will also be split